src/api/app/models/project/remote_url.rb

```diff
@@ -2,11 +2,21 @@ class Project::RemoteURL
   require 'open-uri'
 
   def self.load(remote_project, path)
+    if File.file?('/etc/sysconfig/proxy')
+      proxysettings = Hash[File.read('/etc/sysconfig/proxy').scan(/(\S+)\s*=\s*"([^"]+)/)]
+      Rails.logger.info "var HTTP_PROXY: #{proxysettings.fetch("HTTP_PROXY")} var HTTPS_PROXY: #{proxysettings.fetch("HTTPS_PROXY")} var NO_PROXY: #{proxysettings.fetch("NO_PROXY")}"
+      ENV['http_proxy'] = proxysettings.fetch("HTTPS_PROXY") if proxysettings.has_key?("HTTPS_PROXY")
```

Comment:
```suggestion
      ENV['http_proxy'] = proxysettings['HTTPS_PROXY'] if proxysettings.has_key?("HTTPS_PROXY")
```

Reply:
Fixed as request in stead of fetch(), [] used.